catalog/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.urls import path
+
 from . import views
 
 urlpatterns = [
-    path('home/', views.home, name='home'),
-    path('contacts/', views.contacts, name='contacts'),
-]
\ No newline at end of file
+    path("", views.home, name="home"),
+    path("", views.contacts, name="contacts"),
```

Comment:
Страницу то контактов на отдельном адресе слушать нужно
```suggestion
    path("contacts/", views.contacts, name="contacts"),
```

Reply:
поправил